hack/build-csv-generator.sh

```diff
@@ -15,6 +15,7 @@
 #limitations under the License.
 script_dir="$(cd "$(dirname "$0")" && pwd -P)"
 source "${script_dir}"/common.sh
+setGoInProw
```

Comment:
Maybe you can embed these in the makefile, not 100% sure if that would work though